powershell/internal/ConvertTo-MtMaesterResults.ps1

```diff
@@ -40,8 +40,13 @@ function ConvertTo-MtMaesterResult {
     $tenantName = GetTenantName
     $account = $mgContext.Account
 
-    $currentVersion = ((Get-Module -Name Maester).Version | Select-Object -Last 1).ToString()
```

Comment:
Just a thought: you could leave $currentVersion outside of the check for `Find-Module` and still always show it because `Get-Module` is part of Microsoft.PowerShell.Core.

Reply:
Thanks, implemented!